.github/workflows/nouse_install.yml

```diff
@@ -83,7 +83,11 @@ jobs:
         python3 -c 'import os; print(os.getcwd())'
         python3 -c "import modmesh; print(modmesh.__file__)"
         python3 -c "import _modmesh; print(_modmesh.__file__)"
-        pytest --rootdir=/tmp -v
+        ### The following command is the original commend, and it will fail on pytest == 8.0.0 .
+        ### pytest --rootdir=/tmp -v
+        ### Here is the issue and temporary solution: https://github.com/pytest-dev/pytest/issues/11781  
+        ### The alternative command to solve the issue is ```pytest --rootdir=. -v```.
+        pytest --rootdir=. -v
```

Comment:
I add the comment, including the original command, the modified command, and the source of the issue and solutions.

Reply:
Thanks, the messages are clear.  Could you please follow the comment style to use a single `#` symbol?

Reply:
> Thanks, the messages are clear. Could you please follow the comment style to use a single `#` symbol?

No problem, I modify the inconsistent part and push the modification to the branch.